weave

```diff
@@ -1795,18 +1814,40 @@ case "$COMMAND" in
     # intentionally undocumented since it assumes knowledge of weave
     # internals
     create-bridge)
+        cat 1>&2 <<EOF
```

Comment:
Isn't the theory that no-one will be calling `create-bridge` while docker is running?